### DISCUSSION TITLE
On the relevance between problems

### DISCUSSION THREAD

**Discussion by Yookey** [betalgo/openai#102] [Q&A] [answered]
hi there:
I found that there is no correlation between the problems submitted using the API interface, I can't chat with it continuously, this is different from the feedback obtained by using chatgpt web system.
is this the method I use incorrect?
thank you!

**Comment by kayhantolga** [comment, marked as answer]
https://github.com/betalgo/openai/discussions/89 Hi please have a look at this discussion.  Until OpenAI releases the chatGPT model, either you have to wait or build one by yourself.

**Reply by Yookey** [reply]
Hi kayhantolga:
I am looking forward to the release of the chatGPT model, but it seems that I can only wait.
Thank you for your reply. have a good day :)
